valhalla/baldr/conditional_speed_limit.h

```diff
@@ -0,0 +1,64 @@
+#pragma once
+
+#include <cstdint>
+#include <string>
+
+namespace valhalla {
+namespace baldr {
+
+struct ConditionalSpeedLimit {
```

Comment:
any chance we can simplify this. could you union all the bits that make up the condtion with some member maybe call it `condition_` and then remove the private section entirely. then you should be able to initialize it without a constructor using braces i think like `ConditionalSpeedLimit{speed, condition}`. where we can i just like to have a simple as possible structures. (note i didnt tryy this so there be some gotcha that makes this not work, apologies if so)

Reply:
Well, I can try this
```cpp
struct ConditionalSpeedLimit {
  uint32_t speed_limit() const {
    return speed_limit_;
  }

  uint64_t condition() const {
    return condition_;
  }


  uint64_t condition_: 56;   // TimeDomain meaningful bits
  uint64_t speed_limit_: 8;  // speed limit in KPH
};
```

the all thing will become much smaller, but I'm not sure about this implicit dependency if one day someone decides to use `TimeDomain::spare` bits for something.

Reply:
Actually, I like how it looks like